packages/backend/src/handlers/actions/quests/createCompletion/handler.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+
+import { CreateQuestCompletionOutput, Mutation_RootCreateQuestCompletionArgs } from '../../../../lib/autogen/hasura-sdk';
+import { createCompletion } from './createCompletion';
+
+export const createCompletionHandler = async (
+  req: Request,
+  res: Response,
+): Promise<void> => {
```

Comment:
Maybe in another PR we can create some better abstractions for action handlers. Here's what I've done in the past to have stronger type safety and better dev experience: https://gist.github.com/hammadj/b4c9878b4fd2f9d2eaf98e1543f50ab6

This way we actually only need one top level route for all actions in Express. No need to update in this PR, just wanted to share.

Reply:
Created an issue here: https://github.com/MetaFam/TheGame/issues/356